src/library.js

```diff
@@ -1625,7 +1625,9 @@ LibraryManager.library = {
       if (flag & 256) { // RTLD_GLOBAL
         for (var ident in lib_module) {
           if (lib_module.hasOwnProperty(ident)) {
-            Module[ident] = lib_module[ident];
+            if (ident.substr(0,1) == '_') {
```

Comment:
I wonder if we should also check `typeof` of the property, in that case. But probably not worth it.